src/sql/platform/accounts/common/accountViewModel.ts

```diff
@@ -57,7 +58,7 @@ export class AccountViewModel {
 								},
 								() => { /* Swallow failures at getting accounts, we'll just hide that provider */ });
 					});
-					return Promise.all(promises);
+					return Promise.all(promises).then(accounts => coalesce(accounts));
```

Comment:
I assume this was the behavior fix?

Reply:
Yes, before if the getAccountsForProvider rejected, we were swallowing the error and not returning anything which would become an undefined value in the Promise.all response array.